lib/ansiblelint/utils.py

```diff
@@ -52,6 +53,26 @@
 
 PLAYBOOK_DIR = os.environ.get('ANSIBLE_PLAYBOOK_DIR', None)
 
+INVALID_CONFIG_RC = 2
+ANSIBLE_FAILURE_RC = 3
+
+VERBOSITY_MAP = {
+    0: logging.NOTSET,
+    1: logging.INFO,
+    2: logging.DEBUG
+}
+
+logger = logging.getLogger()
+handler = logging.StreamHandler()
+formatter = logging.Formatter('%(levelname)-8s %(message)s')
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+logger.setLevel(logging.DEBUG)
+
+
+def logger_verbosity(level):
```

Comment:
It should be a verb
```suggestion
def set_logger_verbosity(level):
```

Reply:
Also plz add a docstring

Reply:
@ssbarnea ^ functions perform actions, so their names should hint that some action is done by using a verb. Otherwise, it reads as a property/variable.